setup/cleanup.sh

```diff
@@ -13,16 +13,47 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Bulk delete: if flag `-b` is specified, delete all projects with IDs that
+# match patterns: fourkeys_* or helloworld_*
+
+bulk_delete=0
+
+while getopts ":b" opt; do
+  case ${opt} in
+    b ) bulk_delete=1  ;;
+    \? ) echo "Usage: ./cleanup.sh [-b]" ;;
+  esac
+done
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 [[ -f "$DIR/env.sh" ]] && echo "Importing environment from $DIR/env.sh..." && . $DIR/env.sh
 
-echo "Deleting projects..."
-set -x
-gcloud projects delete ${FOURKEYS_PROJECT}
-gcloud projects delete ${HELLOWORLD_PROJECT}
-gcloud config set project ${PARENT_PROJECT}
-rm env.sh
+if [[ ${bulk_delete} -gt 0 ]]
```

Comment:
NICE!  Now if only there was a way to do this with Datastudio reports!

Reply:
Note: Should document use here https://github.com/GoogleCloudPlatform/fourkeys/blob/main/setup/INSTALL.md#new-google-cloud-projects

Reply:
I added a note.  Feel free to edit it!